src/lib/EventPublisher.js

```diff
@@ -156,12 +167,16 @@ class EventPublisher extends EventEmitter {
         },
         subscriptionTimeout,
       );
-
-      this.once(eventNamePrefix, (): void => clearTimeout(timeout));
+      this.once(eventNamePrefix, () => {
+        clearTimeout(timeout);
+      });
     }
 
     if (once) {
       this.once(eventNamePrefix, listener);
+      this.once(eventNamePrefix, () => {
+        this.unsubscribe(subscriptionID);
```

Comment:
Nope, if the event Fires, every handler is called (and the handler is removed). But i only have to remove the subscription 

```    
this._subscriptionsByID.delete(subscriptionID);
```
 an not call unsubscribe, cause the handler is already removed by ```once```

am i right?